keras/preprocessing/image.py

```diff
@@ -384,8 +384,17 @@ class ImageDataGenerator(object):
         zca_whitening: apply ZCA whitening.
         zca_epsilon: epsilon for ZCA whitening. Default is 1e-6.
         rotation_range: degrees (0 to 180).
-        width_shift_range: fraction of total width, if < 1, or pixels if >= 1.
-        height_shift_range: fraction of total height, if < 1, or pixels if >= 1.
+        width_shift_range: float, 1-D array-like or int
+            height_shift_range: fraction of total height, if < 1, or pixels if >= 1.
+                float: fraction of total width, if < 1, or pixels if >= 1.
+                1-D array-like: random elements from the array.
+                int: integer number of pixels from interval
+                    `(-width_shift_range, +width_shift_range)`
+            With `width_shift_range=2` possible values are integers [-1, 0, +1],
+            same as with `width_shift_range=[-1, 0, +1]`,
+            while with `width_shift_range=1.0` possible values are floats in
+            the interval [-1.0, +1.0).
```

Comment:
Docstring looks super confusing to me. Formatting issues?

Reply:
Removed stray line from the merge